masonry/src/widgets/text_area.rs

```diff
@@ -930,15 +930,20 @@ impl<const EDITABLE: bool> Widget for TextArea<EDITABLE> {
         if !EDITABLE {
             node.set_read_only();
         }
-        self.editor
-            .try_accessibility(
-                ctx.tree_update(),
-                node,
-                || NodeId::from(WidgetId::next()),
-                0.,
-                0.,
-            )
-            .expect("We just performed a layout");
+        let updated = self.editor.try_accessibility(
+            ctx.tree_update(),
+            node,
+            || NodeId::from(WidgetId::next()),
+            0.,
+            0.,
+        );
+
+        let Some(()) = updated else {
+            // FIXME - We should be able to panic here.
+            // The accessibility pass should always run after a layout pass.
+            // See https://github.com/linebender/xilem/issues/1119
+            return;
+        };
```

Comment:
This previously panicking was the only way we knew this was going wrong ... I don't know that we should let this silently get ignored like this.